modules/host/update.go

```diff
@@ -35,24 +34,35 @@ func (h *Host) createStorageProof(obligation contractObligation, heightForProof
 	// Create and send the transaction.
 	id, err := h.wallet.RegisterTransaction(types.Transaction{})
 	if err != nil {
-		return
+		fmt.Println(err)
 	}
 	_, _, err = h.wallet.AddStorageProof(id, sp)
 	if err != nil {
-		return
+		fmt.Println(err)
 	}
 	t, err := h.wallet.SignTransaction(id, true)
 	if err != nil {
-		return
+		fmt.Println(err)
```

Comment:
oh right. There needs to be a defer statement that deletes the obligation. I forgot to implement that.